### PR TITLE
Added authentication and roles section.

### PR DESCRIPTION
**Pull request by MorganEPatch**
Fixes issue #24.

### CODE REVIEW

**Comment by Kennric** [comment]
This looks correct to me. +1, please get another +1

File: source/draft_api.rst [hunk 1]
```diff
@@ -297,3 +297,41 @@ If the object does not exist, the API will return an Object Not Found error
...
 docs).
+
+-----------------------
+
+Authorization and Roles
+-----------------------
+
+Each timesync user can be of one of two roles: user, and admin. In addition, each user
+has a role within each project to which they belong: member, data viewer, and project
+manager. Each role is cumulative: that is, a data viewer has all of the abilities that a
```
Comment (tschuy) [inline source/draft_api.rst]: do we want viewers to be able to write? that seems like [Grade A Misleading](http://i.imgur.com/XHMhxcr.png), and a Very Bad Idea.

File: source/draft_api.rst [hunk 2]
```diff
@@ -297,3 +297,41 @@ If the object does not exist, the API will return an Object Not Found error
...
 docs).
+
+-----------------------
+
+Authorization and Roles
+-----------------------
+
+Each timesync user can be of one of two roles: user, and admin. In addition, each user
+has a role within each project to which they belong: member, data viewer, and project
+manager. These roles exist independently, and are defined by their permissions: a user
```
Comment (tschuy) [inline source/draft_api.rst]: this seems like a good place for listification: https://github.com/osuosl/docs/pull/138/files#diff-4036c50de2e6cbdbdb43d66bacb718ffR10

File: source/draft_api.rst [hunk 3]
```diff
@@ -297,3 +297,48 @@ If the object does not exist, the API will return an Object Not Found error
 
 In case of any other error, the API will return a Server Error (see error
 docs).
+
+-----------------------
+
+Authorization and Roles
+-----------------------
+
+Each timesync user can be of one of two roles: user, and admin. In addition, each user
```
Comment (tschuy) [inline source/draft_api.rst]: Perhaps explain admin role?

**Comment by tschuy** [comment]
Who can create/modify projects?

**Comment by tschuy** [comment]
+1

**Comment by Kennric** [comment]
+1

**Pull request merged**